tests/fixtures/expected/blocks/nodes/figure.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="utf-8" />
+</head>
+<body>
+
+<figure>
+    <!-- REMOVE: src is empty just because the test doesn't utilize the full stack, with CopyImagesListener -->
```

Comment:
I really wanted to document why the `src=` is empty. Making it ACTUALLY work would be a lot more work to make the test much more complex, and I don't think it's worth it (I'm pretty sure the images and image paths are tested elsewhere).